packages/lib-classifier/src/components/Classifier/components/ImageToolbar/components/RotateButton/RotateButtonContainer.js

```diff
@@ -1,36 +1,43 @@
 import PropTypes from 'prop-types'
-
-import { withStores } from '@helpers'
+import { useStores } from '@hooks'
```

Comment:
I notice in https://local.zooniverse.org:3000/projects/nora-dot-eisner/planet-hunters-tess/classify/workflow/11235?env=production the rotate button is unexpectedly shown.

I'm not sure, but maybe it's the missing observer will fix.

Reply:
Good catch! I didn't correctly pass all of the subject viewer store props in `useStores`. I'll correct that before merging this PR.